src/encoded/schemas/experiment_series.json

```diff
@@ -47,8 +46,17 @@
             "default": "1"
         },
         "related_datasets": {
```

Comment:
can't you still use the seiries properties with the $ref, and only specify the "minItem" in the overwritten related_datasets? I am not sure it will work though.

Reply:
I think other series use this related_datasets property such as MatchedSet and OrganismDevelopmentSeries, which have empty lists. I don't think I can remove the "default: []" from series properties? Correct me if I'm wrong

Reply:
I tested out using it as a $ref with "minItem: 1" but the default [] still remained, so I will have to go back to my original approach.